demisto_sdk/commands/generate_yml_from_python/README.md

```diff
@@ -0,0 +1,1510 @@
+# Generate YML from python
+Generate YML file from python code that includes special syntax.
+The output file name will be the same as the python code with the .yml ending instead of .py.
```

Comment:
```suggestion
Generate YML file from Python code that includes special syntax.
The output file name will be the same as the Python code with the `.yml` extension instead of `.py`.
```